Modules/hmacmodule.c

```diff
@@ -491,7 +388,7 @@ narrow_hmac_hash_kind(hmacmodule_state *state, HMAC_Hash_Kind kind)
  * Otherwise, this sets an appropriate exception and returns -1.
  */
 static int
-_hacl_convert_errno(hacl_errno_t code, PyObject *algorithm)
+_hacl_convert_errno(hacl_errno_t code)
```

Comment:
I'll isolate that change as well. It's actually un-necessary to pass the algorithm to this helper because the traceback would indicate which class caused the issue.